### PR TITLE
Adding detail to CUPED docs

### PR DESCRIPTION
**Pull request by schmit**
Adding more details and a plot to illustrate the benefits of CUPED to our docs. Feel free to propose more changes!

### CODE REVIEW

**Review by evanmiller — APPROVED** [review]

File: docs/interpreting-experiments/cuped.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
 # Controlled-experiment Using Pre-Existing Data (CUPED)
-CUPED is a statistical tool that leverages pre-existing data to reduce the variance of an A/B experiment. If data is music, CUPED is a pair of noise canceling headphones, allowing you to notice a more pronounced pattern in the data despite a lower volume or strong external noise.
+The precision of experiment results (the width of confidence intervals) depends on the variance in of the metrics we are measuring. One way to improve the precision of experiments is to gather more data, as the variance goes down as we gather more data; this obviously means that it takes longer to run an experiment. 
+
+However, there are options beyond waiting to gather more data, and all these options address reducing the variance in the metrics we are measuring directly. One particular flexible and powerful method is known as CUPED. CUPED is a statistical tool that leverages pre-existing data to reduce the variance of an A/B experiment. You can think of CUPED as a pair of noise cancelling headphones: it uses data gathered prior to the experiment to understand the ambient noise, allowing you to notice a more pronounced pattern in the data.
```
Comment (bkfunk) [inline docs/interpreting-experiments/cuped.md:4]: I think the first sentence is confusing. How about "However, there are more direct ways to reduce variance, beyond just waiting for more data to show up."

Also, "One particularLY flexible and powerful method"? I would also spell out the acronym: "known as CUPED, which stands for Controlled...[etc]."

I would also change "pre-existing": what's relevant is that it is data from _before_, or more properly _outside_ the experiment. How about something like "Standard experiment analysis involves comparing metric data from subjects exposed to a treatment to that of a control group -- all the data is gathered _during_ the experiment. But most companies know something about their users beyond what they did during the experiment, most prominently metric data from _before_ the experiment started. CUPED leverages this data from outside the experiment in order to control for some of the variance in metrics that comes from randomly picking variants for each subject; in a standard experiment, you might end up with one variant having more active users just by random chance, while CUPED reduces the effect of this random variation by _controlling for_ the different activity levels across different variants."

File: docs/interpreting-experiments/cuped.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 
 Eppo uses CUPED to get more accurate estimates of treatment effect, which should lead to narrower confidence intervals.
 
 ## Overview
-When utilizing CUPED, Eppo automatically ingests event data of subjects in the 30 days prior to experiment start across specified metrics. CUPED works best for experiments with long-time users for whom many pre-experiment data points exist. It is less effective for newer users.
+When utilizing CUPED, Eppo automatically ingests event data of subjects in the 30 days prior to experiment start across metrics that are added to your experiment. Eppo then uses a ridge regression model to predict subject level outcomes for each metric. Relationships between every metric are taken into account to make predictions for other metrics. These predictions are then used to get more accurate effect estimates, reducing the variance and hence leading to better confidence intervals. In particular, our approach allows you to leverage variables with rich datasets to reduce variance for sparser data where pre-experiment datapoints might be lacking (for example, retention and conversion).
```
Comment (bkfunk) [inline docs/interpreting-experiments/cuped.md:12]: I think the first sentence is awkward; how about "When you choose to enable CUPED, Eppo will ingest aggregated [? is this true] data about each subject's events (for each metric added to the experiment) for the 30 days before they joined the experiment."

In the next sentence, it should be "subject-level" (hyphen).

For the sentence starting "Relationships", I think it's a bit unclear...how about "For each metric, we predict the outcomes using _every_ available metric."

Then for the last sentence, I think it's confusing that we compare "rich datasets" to "sparser data"...why not use "features" or "metrics"? "In particular, our approach allows you to leverage dense metrics (where subjects have a wide variety of values) to reduce variance for sparser metrics, such as retention and conversion, where only a small fraction of subjects have pre-experiment events.

File: docs/interpreting-experiments/cuped.md [hunk 3]
```diff
@@ -1,15 +1,20 @@
 # Controlled-experiment Using Pre-Existing Data (CUPED)
-CUPED is a statistical tool that leverages pre-existing data to reduce the variance of an A/B experiment. If data is music, CUPED is a pair of noise canceling headphones, allowing you to notice a more pronounced pattern in the data despite a lower volume or strong external noise.
+The precision of experiment results (the width of confidence intervals) depends on the variance in of the metrics we are measuring. One way to improve the precision of experiments is to gather more data, as the variance goes down as we gather more data; this obviously means that it takes longer to run an experiment.
```
[the remark this answers is not in the document]
Comment (schmit) [reply docs/interpreting-experiments/cuped.md:2]: Most customers aren't in the data regime where they afford to enroll less than 100% of traffic, so not sure if it's worth highlighting.

File: docs/interpreting-experiments/cuped.md (hunk 2 above)
Comment (schmit) [reply docs/interpreting-experiments/cuped.md:12]: Great stuff; re last comment I think it makes most sense to just remove the last sentence

**Pull request merged**